onnx/test/shape_inference_test.py

```diff
@@ -1097,6 +1097,36 @@ def test_split_negative_axis(self):  # type: () -> None
         self._assert_inferred(graph, [make_tensor_value_info('y', TensorProto.FLOAT, (2, 2)),
                                       make_tensor_value_info('z', TensorProto.FLOAT, (2, 2))])
 
+    def test_split_with_split_attribute(self):  # type: () -> None
```

Comment:
can you please add 2 more tests - 

1) One with split attribute is missing and the default 'equal' split occurs for each output
2) One with (valid) negative axis

Reply:
Isn't the existing test case, `test_split_negative_axis` enough to cover these two cases? Do you want me to split it into two tests?

Reply:
You are right. :) I did not notice the other test.